packages/plugins/i18n/server/src/bootstrap.ts

```diff
@@ -14,6 +14,30 @@ const registerModelsHooks = () => {
       await getService('permissions').actions.syncSuperAdminPermissionsWithLocales();
     },
   });
+
+  const i18nModelUIDs = Object.values(strapi.contentTypes)
+    .filter((contentType) => getService('content-types').isLocalizedContentType(contentType))
+    .map((contentType) => contentType.uid);
+
+  strapi.documents.use(async (context, next) => {
+    // @ts-expect-error ContentType is not typed correctly on the context
+    const schema = context.contentType;
+
+    if (!['create', 'update'].includes(context.action)) {
```

Comment:
You will also need to react to the discardDraft action as we will need to re-update all the other locales (unless discardDraft calls update directly ? )

Reply:
same for publish